[Ready] Sushi!

Trilbyspaceclone opened the pull request:
[Changelogs]
Adds a few types of sushi!
Funa Hosomaki - role of rice and fish
Funa Nigiri - Rice with fish atop
Funa Nigiri - Well made
Onigiri -Rice ball
Tobiko - fish eggs rolled in seaweed
Each sushi needs soy souse and rice, as well as some using Kudzu for seaweed
Do to each Sushi eating bowls, If they use them all then they can ask cargo for more...
![image](https://user-images.githubusercontent.com/30435998/55691957-57291f00-5971-11e9-9ac5-2f96be384c83.png)
![image](https://user-images.githubusercontent.com/30435998/55691972-8049af80-5971-11e9-9cde-c11bcc4f2a25.png)

[why]
More food is always nice, carp and kuz are to rarely used as well giving more options into the round for food later on!

Trilbyspaceclone committed:
Merge pull request #31 from Citadel-Station-13/master

Update

Trilbyspaceclone committed:
Delete food.dmi

Trilbyspaceclone committed:
Add files via upload

Trilbyspaceclone committed:
properly

Trilbyspaceclone committed:
not the correct one

Trilbyspaceclone committed:
Update snacks_other.dm

Trilbyspaceclone committed:
Update snacks_meat.dm

Trilbyspaceclone committed:
Update snacks_other.dm

Trilbyspaceclone committed:
Update recipes_meat.dm

Trilbyspaceclone committed:
Update recipes_misc.dm

Trilbyspaceclone committed:
Update recipes_meat.dm

Trilbyspaceclone committed:
Update recipes_misc.dm

Trilbyspaceclone committed:
Update recipes_meat.dm

Trilbyspaceclone committed:
Update recipes_meat.dm

Trilbyspaceclone committed:
15 is 3 x 5 more lucky?

Trilbyspaceclone committed:
Update recipes_misc.dm

Trilbyspaceclone committed:
Update recipes_meat.dm

Trilbyspaceclone committed:
Update snacks_meat.dm

Trilbyspaceclone committed:
reblance

Trilbyspaceclone committed:
reblance

Trilbyspaceclone committed:
Update recipes_meat.dm

Trilbyspaceclone committed:
Update recipes_misc.dm

Trilbyspaceclone committed:
Update drinnerware.dm

Poojawa commented on code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm:
```diff
@@ -81,6 +81,37 @@
+	reqs = list(
+		/datum/reagent/consumable/soysauce = 10,
+		/obj/item/reagent_containers/food/snacks/salad/ricebowl = 2,
+		/obj/item/reagent_containers/food/snacks/carpmeat = 5
```
5 fillets??

Trilbyspaceclone replied:
Yep Yep!
You taking the core of each one and seamlessly layering them for the best and freshens tastes as well as soaking them in a thin layer of soy sauce!

Ragolution replied:
i agree with pooj

Poojawa commented on code/modules/food_and_drinks/food/snacks_meat.dm:
```diff
@@ -54,6 +54,43 @@
+	icon_state = "sushie_pro"
+	bonus_reagents = list("nutriment" = 2, "vitamin" = 2)
+	list_reagents = list("nutriment" = 8, "vitamin" = 2)
+	bitesize = 1
```
five carp fillets for 1 bite and 10 nutriment. 

seems uh, a bit off?

Trilbyspaceclone replied:
1 bite means it takes 1 of each nutriment per bite
Its not ment to be eaten all at once!

Ragolution replied:
Bite size still needs to be changed. The carp fillets, on their own, are a size 3 bite of 7-9. This is way smaller and counter-intuitive. Considering the fact that sushi is a bite-size piece of food, this makes no sense.

Ragolution commented on code/modules/vending/drinnerware.dm:
```diff
@@ -13,7 +13,7 @@
 		            /obj/item/reagent_containers/food/condiment/saltshaker = 5,
 		            /obj/item/reagent_containers/food/condiment/peppermill = 5,
-		            /obj/item/reagent_containers/glass/bowl = 20)
+		            /obj/item/reagent_containers/glass/bowl = 50)
```
Make it return the empty bowls. Don't do this because you don't know how.

Ragolution replied:
You don't start with enough rice to use all of these bowls to begin with, nevermind the fact they should be re-usable.

Trilbyspaceclone replied:
I legit tryed. I tryed lists, I tryed making them be the trash item. Im just to cat brained to make it work

Trilbyspaceclone replied:
If you want to make a way for crafting to become list friendly and able to work. That would be grand but as it is, best thing I can do is make bowls more easy to make. Let alone the fact that the dish drive treats them as trash.
Ideally the Dish Drive makes new dishes rather then just teles them into trash cans

Ragolution commented on code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm:
```diff
@@ -81,6 +81,37 @@
+	subcategory = CAT_MEAT
+
+/datum/crafting_recipe/food/sushie_pro
+	name = "Well made Funa Nigiri"
```
not capitalized correctly

Ragolution replied:
why does this differentiation even exist?

Trilbyspaceclone replied:
Its capitalized as others, fits with the file and other formatting.
Two are for more options, and such.

Ragolution replied:
Literally no reason.

Trilbyspaceclone replied:
Its capitalized as others, fits with the file and other formatting.
Two are for more options, and such.

Ragolution commented on code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm:
```diff
@@ -81,6 +81,37 @@
+		/datum/reagent/consumable/soysauce = 3,
+		/obj/item/reagent_containers/food/snacks/salad/ricebowl = 1,
+		/obj/item/reagent_containers/food/snacks/carpmeat = 2,
+		/obj/item/reagent_containers/food/snacks/grown/kudzupod = 3,
```
So these'll be used about once a year since kudzu spawns are terribly rare and no botanists ever play with it because they know they'll get absolutely destroyed for self-antagging.

Trilbyspaceclone replied:
Kuz is the flat out easiest weed to use, and its get-able 3 mins of round start

Ragolution commented on code/modules/food_and_drinks/food/snacks_other.dm:
```diff
@@ -207,6 +207,16 @@
+	name = "Tobiko"
+	desc = "Spider eggs wrapped in a thin salted Kudzu pod"
+	icon = 'modular_citadel/icons/obj/food/food.dmi'
+	icon_state = "sushie_egg"
```
Is this a typo?

Ragolution commented on code/modules/food_and_drinks/food/snacks_meat.dm:
```diff
@@ -54,6 +54,43 @@
+	name = "Funa Nigiri"
+	desc = "A well prepared pice of the best of the carp fillet placed on rice. Looks fancy and fresh!"
+	icon = 'modular_citadel/icons/obj/food/food.dmi'
+	icon_state = "sushie_pro"
```
Is this a typo?

Ragolution commented on code/modules/food_and_drinks/food/snacks_meat.dm:
```diff
@@ -54,6 +54,43 @@
+	name = "Funa Nigiri"
+	desc = "A pice of carp lightly placed on some rice."
+	icon = 'modular_citadel/icons/obj/food/food.dmi'
+	icon_state = "sushie_adv"
```
Is this a typo?

Ragolution commented on code/modules/food_and_drinks/food/snacks_meat.dm:
```diff
@@ -54,6 +54,43 @@
+	tastes = list("fish" = 1, "rice" = 1, "salt" = 1)
+	foodtype = MEAT | VEGETABLES
+
+/obj/item/reagent_containers/food/snacks/sushie_pro
```
Is this a typo?

Ragolution commented on code/modules/food_and_drinks/food/snacks_meat.dm:
```diff
@@ -54,6 +54,43 @@
+	name = "Funa Hosomaki"
+	desc = "A small cylindrical filled with rice and fish."
+	icon = 'modular_citadel/icons/obj/food/food.dmi'
+	icon_state = "sushie_basic"
```
Is this a typo?

Ragolution commented on code/modules/food_and_drinks/food/snacks_meat.dm:
```diff
@@ -54,6 +54,43 @@
 
+/obj/item/reagent_containers/food/snacks/sushie_basic
+	name = "Funa Hosomaki"
+	desc = "A small cylindrical filled with rice and fish."
```
Is this a typo?

Trilbyspaceclone replied:
fixed?

Ragolution commented on code/modules/food_and_drinks/food/snacks_meat.dm:
```diff
@@ -54,6 +54,43 @@
+
+/obj/item/reagent_containers/food/snacks/sushie_adv
+	name = "Funa Nigiri"
+	desc = "A pice of carp lightly placed on some rice."
```
Is this a typo?

Trilbyspaceclone replied:
What part?

Ragolution replied:
"PICE"

Ragolution commented on code/modules/food_and_drinks/food/snacks_meat.dm:
```diff
@@ -54,6 +54,43 @@
+
+/obj/item/reagent_containers/food/snacks/sushie_pro
+	name = "Funa Nigiri"
+	desc = "A well prepared pice of the best of the carp fillet placed on rice. Looks fancy and fresh!"
```
Is this a typo?

Trilbyspaceclone replied:
What part?

Ragolution replied:
"PICE"

Ragolution commented on code/modules/food_and_drinks/food/snacks_meat.dm:
```diff
@@ -54,6 +54,43 @@
+	list_reagents = list("nutriment" = 4)
+	bitesize = 1
+	filling_color = "#F2EEEA" //rgb(242, 238, 234)
+	tastes = list("fish" = 1, "rice" = 1, "salt" = 1)
```
Having eaten sushi before, I am completely in doubt of the amount of fishy taste being equal to salty.

Ragolution commented on code/modules/food_and_drinks/food/snacks_other.dm:
```diff
@@ -207,6 +207,16 @@
+	icon_state = "sushie_egg"
+	list_reagents = list("nutriment" = 6, "vitamin" = 2)
+	filling_color = "#FF3333" // R225 G051 B051
+	tastes = list("seaweed" = 1, "salty" = 2)
```
but there's no seaweed in it?

Trilbyspaceclone replied:
Kuz pods is the idea behind it, weed like taste + salty 1 + toxins from the eggs was the idea

Ragolution commented on code/modules/food_and_drinks/food/snacks_other.dm:
```diff
@@ -569,3 +579,12 @@
+
+/obj/item/reagent_containers/food/snacks/riceball
+	name = "Onigiri"
+	desc = "A ball of rice with some light salt and a wrap of Kudzu skin."
```
Typically onigiri have no salt but are instead filled with a rice-topping like umeboshi or furikake

Trilbyspaceclone replied:
Wikia quote - Traditionally, an onigiri is filled with pickled ume (umeboshi), salted salmon, katsuobushi, kombu, tarako, or any other salty or sour ingredient as a natural preservative.

Ragolution commented on code/modules/food_and_drinks/food/snacks_other.dm:
```diff
@@ -569,3 +579,12 @@
 	tastes = list("chocolate" = 1)
 	foodtype = JUNKFOOD | SUGAR
+
+/obj/item/reagent_containers/food/snacks/riceball
```
Why not just make a /food/snacks/sushi and make these all subtypes of that? You seem to be rewriting code here a number of times for no reason.

Trilbyspaceclone replied:
Their like this to fit in with other food groups that of grains like rice pudding or such

Ragolution replied:
Use inheritance.

Ragolution commented on code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm:
```diff
@@ -81,6 +81,37 @@
+		/obj/item/reagent_containers/food/snacks/carpmeat = 2,
+		/obj/item/reagent_containers/food/snacks/grown/kudzupod = 3,
+	)
+	result = /obj/item/reagent_containers/food/snacks/sushie_basic
```
Is this a typo?

Ragolution commented on code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm:
```diff
@@ -81,6 +81,37 @@
+	result = /obj/item/reagent_containers/food/snacks/sushie_basic
+	subcategory = CAT_MEAT
+
+/datum/crafting_recipe/food/sushie_adv
```
Is this a typo?

Ragolution commented on code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm:
```diff
@@ -81,6 +81,37 @@
+		/obj/item/reagent_containers/food/snacks/salad/ricebowl = 1,
+		/obj/item/reagent_containers/food/snacks/carpmeat = 1
+	)
+	result = /obj/item/reagent_containers/food/snacks/sushie_adv
```
Is this a typo?

Ragolution commented on code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm:
```diff
@@ -81,6 +81,37 @@
+		/obj/item/reagent_containers/food/snacks/salad/ricebowl = 2,
+		/obj/item/reagent_containers/food/snacks/carpmeat = 5
+	)
+	result = /obj/item/reagent_containers/food/snacks/sushie_pro
```
Is this a typo?

Ragolution commented on code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm:
```diff
@@ -93,6 +124,16 @@
 	result = /obj/item/reagent_containers/food/snacks/spidereggsham
 	subcategory = CAT_MEAT
 
+/datum/crafting_recipe/food/sushie_egg
```
Is this a typo?

Trilbyspaceclone replied:
E is at the end for labeling. Its a active chose for later ideas and improvements

Ragolution replied:
that doesn't make sense

Ragolution commented on code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm:
```diff
@@ -93,6 +124,16 @@
 	subcategory = CAT_MEAT
 
+/datum/crafting_recipe/food/sushie_egg
+	name = "Tobiko"
```
If you included Tobiko why not include tamago, using eggs you'll actually have access to every round?

Trilbyspaceclone replied:
Their not ment to be round start. Its Exotic

Ragolution commented on code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm:
```diff
@@ -81,6 +81,37 @@
+	reqs = list(
+		/datum/reagent/consumable/soysauce = 5,
+		/obj/item/reagent_containers/food/snacks/salad/ricebowl = 1,
+		/obj/item/reagent_containers/food/snacks/carpmeat = 1
```
tbh im waxing but this would be much better if we actually had working aquariums/aquaculture

Trilbyspaceclone replied:
We used to, but all of the fish were uneatable!

Ragolution commented on code/modules/food_and_drinks/food/snacks_meat.dm:
```diff
@@ -54,6 +54,43 @@
+	icon_state = "sushie_adv"
+	bonus_reagents = list("vitamin" = 2)
+	list_reagents = list("nutriment" = 6)
+	bitesize = 1
```
Considering the fact that sushi is a bite-size piece of food, this makes no sense.

Ragolution commented on code/modules/food_and_drinks/food/snacks_meat.dm:
```diff
@@ -54,6 +54,43 @@
+	icon_state = "sushie_basic"
+	bonus_reagents = list("vitamin" = 2)
+	list_reagents = list("nutriment" = 4)
+	bitesize = 1
```
Granted that Sushi is, by definition, bite-sized, I don't feel this is appropriate.

Trilbyspaceclone committed:
grammar and "Bitsize fix"

Trilbyspaceclone committed:
Update snacks_other.dm

Traditionally, an onigiri is filled with pickled salted salmon, katsuobushi, kombu, tarako, or any other salty or sour ingredient as a natural preservative.

Trilbyspaceclone committed:
grammar and lowering of Well made 5 -> 2

Trilbyspaceclone committed:
Update drinnerware.dm

Trilbyspaceclone committed:
grammar and balance

Trilbyspaceclone committed:
tastes

Trilbyspaceclone committed:
fixed more grammar
